lib/mayaUsd/ufe/UsdSetXformOpUndoableCommandBase.h

```diff
@@ -73,6 +74,9 @@ class UsdSetXformOpUndoableCommandBase : public Ufe::SetVector3dUndoableCommand
 
     PXR_NS::UsdTimeCode writeTime() const { return _writeTime; }
 
+    // Retrieve the USD prim affected by the command.
+    virtual PXR_NS::UsdPrim getPrim() const;
```

Comment:
The reason to make it virtual is that sub-classes can make getting the UsdPrim more efficient.